inferno-vc/src/Inferno/VersionControl/Operations.hs

```diff
@@ -340,87 +339,63 @@ fetchVCObjectHistory h = do
     head_h <- fetchCurrentHead h
     let head_fp = storePath </> "heads" </> show head_h
     preds <- readVCObjectHashTxt head_fp
-    pure $ head_h : preds
```

Comment:
`preds` is stored in file as oldest-to-newest so this line adds `head_h` to the wrong end of the history.